src/lib.rs

```diff
@@ -31,14 +31,21 @@ pub mod config;
 /// Runs a mosaic node with the given configuration.
 /// Prints all accounts of the origin blockchain to std out.
 pub fn run(config: &Config) -> Result<(), Box<Error>> {
-    let blockchain = Blockchain::new(&Kind::Eth, config.origin_endpoint());
+    let origin = Blockchain::new(&Kind::Eth, config.origin_endpoint());
+    let auxiliary = Blockchain::new(&Kind::Eth, config.auxiliary_endpoint());
 
-    let accounts = blockchain.get_accounts();
+    let origin_accounts = origin.get_accounts();
+    let auxiliary_accounts = auxiliary.get_accounts();
 
-    println!("Accounts:");
-    for account in accounts {
+    println!("Origin accounts:");
```

Comment:
should we remove these `println`?

Reply:
It will be removed in a later PR, it is just example code at this stage.